config/desktop/focal/environments/cinnamon/config_base/packages

```diff
@@ -103,6 +103,7 @@ libwmf0.2-7-gtk
 libxcursor1
 lightdm
 lightdm-settings
+lm-sensros
```

Comment:
should this be lm-sensors?

Reply:
And also this should probably go to the CLI packages - to all distros.